### PR TITLE
Replaces Tricordrazine in survival medipens with Regenerative Jelly

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Replaces Tricordrazine in survival medipens with Regenerative Ooze

## Why It's Good For The Game
 This PR achieved the goal of making basic healing medicines less effective for treating all types of injirues : https://github.com/BeeStation/BeeStation-Hornet/pull/7936
 However, it did not account for the survival medipens heavily relying on Tricordrazine for the healing. After a talk we had in the discord a while back we settled on Regenerative Ooze being a good replacement but then we kind of forgot about it, so I'm only making it just now.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: Replaced Tricordrazine in survival medipens with Regenerative Jelly
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Whats the healing rate different between regenerative ooze and tricordazine. I'd rather have something long lasting, slow but powerful than something that heals you up extremely quickly

**Comment by CydiaLamiales** [comment]
> Whats the healing rate different between regenerative ooze and tricordazine. I'd rather have something long lasting, slow but powerful than something that heals you up extremely quickly

Tricodraizine heals 1 of each basic damage per tick but has the new calculation of having a chance to heal, with it getting lower the more hurt you are.
Regen ooze is a constant 0.5 heal of every damage type.

**Comment by PowerfulBacon** [comment]
What are the actual total numbers of healing this provides and the timespan difference though?

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -225,7 +225,7 @@
 	item_state = "survpen"
 	volume = 57
 	amount_per_transfer_from_this = 57
-	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/tricordrazine = 15, /datum/reagent/medicine/epinephrine = 10, /datum/reagent/medicine/lavaland_extract = 2, /datum/reagent/medicine/omnizine = 5)
+	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/regen_ooze = 15, /datum/reagent/medicine/epinephrine = 10, /datum/reagent/medicine/lavaland_extract = 2, /datum/reagent/medicine/omnizine = 5)
```
Comment (CydiaLamiales) [inline code/modules/reagents/reagent_containers/hypospray.dm]:
```suggestion
	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/regen_jelly = 15, /datum/reagent/medicine/epinephrine = 10, /datum/reagent/medicine/lavaland_extract = 2, /datum/reagent/medicine/omnizine = 5)
```

**Comment by PowerfulBacon** [comment]
Whats the actual numbers though so I know how the pen is going to function after these changes?

**Comment by PowerfulBacon** [comment]
Over what time period though? Find the datum for the reagent and post the metabolisation rate and the amount it heals per tick. If you are changing the things in an item, then we need to know how that its going to change things, I don't want to add an item that lets anyone instantly get 20 health back within 1 second mid-fight.

**Comment by CydiaLamiales** [comment]
> Over what time period though? Find the datum for the reagent and post the metabolisation rate and the amount it heals per tick. If you are changing the things in an item, then we need to know how that its going to change things, I don't want to add an item that lets anyone instantly get 20 health back within 1 second mid-fight.

Tricordrazine's rate is 1.2 , Jelly's is 0.4

**Review by itsmeow — APPROVED** [review]
Appears to return it to former effectiveness.

**Comment by EvilDragonfiend** [comment]
It might need to reconsider Ruko's upcoming med PR.

**Comment by CydiaLamiales** [comment]
> It might need to reconsider due to Ruko's upcoming med PR.

That doesn't have metabolytes for Regenerative Jelly and, in fact, makes Trico even worse at healing.

**Pull request merged**